src/components/ConnectButtonAvatar.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FunctionComponent } from 'react';
+
+import { MetaMaskIcon } from './icons';
+
+export type ConnectButtonAvatarProps = {
+  isConnected: boolean;
+  isFallback: boolean;
+};
+
+export const ConnectButtonAvatar: FunctionComponent<
+  ConnectButtonAvatarProps
+> = ({ isConnected, isFallback }) => {
+  const width = { base: '0', sm: '1.3rem' };
```

Comment:
This can be inlined now since it's only used in one place.

Reply:
resolved